compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmCodeGenUtil.java

```diff
@@ -507,30 +511,68 @@ public static boolean isBallerinaBuiltinModule(String orgName, String moduleName
         return orgName.equals("ballerina") && moduleName.equals("builtin");
     }
 
-    public static void generateBbInstructions(MethodVisitor mv, LabelGenerator labelGen, JvmInstructionGen instGen,
-                                              int localVarOffset, AsyncDataCollector asyncDataCollector,
-                                              String funcName,
-                                              BIRNode.BIRBasicBlock bb) {
+    public static BirScope generateBbInstructions(MethodVisitor mv, LabelGenerator labelGen, JvmInstructionGen instGen,
+            int localVarOffset, AsyncDataCollector asyncDataCollector,
+            String funcName,
+            BIRNode.BIRBasicBlock bb, Set<BirScope> visitedScopesSet, BirScope lastScope) {
```

Comment:
not sure your formatting setting is correct here? the new line should start from the same column level as the previous line for method arguments.

Reply:
I guess somethings wrong with my ide formatter. I'll check all the instances again